### PR TITLE
style: remove react bootstrap fade component

### PR DESCRIPTION
**Pull request by nikolagigic**
### SUMMARY
Replacing Bootstrap component with custom Emotion/CSS component with same behaviour in effort to move away from Bootstrap.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
https://www.loom.com/share/e1ba9e7671c34581858043bfca669ea1
After:
https://www.loom.com/share/7228d0769f3247bba5aedec407d7f3bc

### TEST PLAN
	1.	Log into Superset
	2.	Navigate to SQL Lab > SQL Editor (/superset/sqllab)
	3.	Query a table
	4.	Hover over Table Element on the left side of the screen

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/common/components/Fade.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * under the License.
+ */
+import { styled } from '@superset-ui/core';
+
+type FadeProps = { hovered: boolean };
+
+const Fade = styled.div<FadeProps>`
+  transition: all ${({ theme }) => theme.transitionTiming}s;
+  opacity: ${props => (props.hovered ? 1 : 0)};
+`;
```
Comment (rusackas) [inline superset-frontend/src/common/components/Fade.tsx:26]: This works... but would there be a downside to a simpler approach of just making it:
```
  transition: opacity ${({ theme }) => theme.transitionTiming}s;
  opacity: 0;
  &:hover {
    opacity: 1
  }
```
then, I _think_ you could go through and remove all the code regarding the `hovered` state and prop.

Comment (nikolagigic) [reply superset-frontend/src/common/components/Fade.tsx:26]: With this approach **<Fade>** component would only fade in/out if we hover over the header which is not the previous behaviour we had. The previous behaviour would trigger fade in/out when we hover over **TableElement** component (header and body).

**Comment by junlincc** [comment]
@nikolagigic thanks for the PR! if it's ok with you, going forward, can we attach screenshot or gif on the PRs, instead of links to video? thanks! 🙏

**Comment by rusackas** [comment]
> @nikolagigic thanks for the PR! if it's ok with you, going forward, can we attach screenshot or gif on the PRs, instead of links to video? thanks! 🙏

I think there are a thousand options, but I've had good luck with Giphy Capture.

**Comment by nikolagigic** [comment]
@junlincc @rusackas will do!

**Review by rusackas — APPROVED** [review]
LGTM!

There's a bit of an ongoing discussion as to where new components like this should go. This seems like a reasonable place for now, but at some point I want to make things a little more consistent, with components each having a folder with their own test file(s), a Storybook file, and the component itself. That can be a task for another day... this one is obviously pretty light/utilitarian. I appreciate the test!

**Pull request merged**